src/modules/common/dto/pagination.dto.ts

```diff
@@ -17,6 +17,10 @@ export class PaginationDto {
   readonly hasMore: boolean;
 
   constructor(values) {
+    if (!values) {
+      return;
+    }
```

Comment:
I had to add this early return since `ValidationPipe` instantiates a new `MentorFiltersDto` object without passing any values.

Reply:
If you comment out the early return and run the tests, you'll notice the following error.

```
TypeError: Cannot read property 'page' of undefined
    at new PaginationDto (find-a-mentor-api/src/modules/common/dto/pagination.dto.ts:22:15)
    at new MentorFiltersDto (find-a-mentor-api/src/modules/common/dto/mentorfilters.dto.ts:6:1)
    at TransformOperationExecutor.Object.<anonymous>.TransformOperationExecutor.transform (find-a-mentor-api/src/TransformOperationExecutor.ts:123:32)
    at ClassTransformer.Object.<anonymous>.ClassTransformer.plainToClass (find-a-mentor-api/src/ClassTransformer.ts:43:25)
    at Object.plainToClass (find-a-mentor-api/src/index.ts:37:29)
    at ValidationPipe.transform (find-a-mentor-api/node_modules/@nestjs/common/pipes/validation.pipe.js:55:41)
    at find-a-mentor-api/node_modules/@nestjs/core/pipes/pipes-consumer.js:15:33
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at find-a-mentor-api/node_modules/@nestjs/core/router/router-execution-context.js:137:31
    at async Promise.all (index 0)
    at pipesFn (find-a-mentor-api/node_modules/@nestjs/core/router/router-execution-context.js:134:13)
    at find-a-mentor-api/node_modules/@nestjs/core/router/router-execution-context.js:35:30
    at find-a-mentor-api/node_modules/@nestjs/core/router/router-execution-context.js:44:28
    at find-a-mentor-api/node_modules/@nestjs/core/router/router-proxy.js:8:17
    ```

Reply:
That's a bummer.  Nice work on figuring that out!